tests/test_filter_datasets.py

```diff
@@ -294,3 +341,109 @@ def test_stratified_crossval_splits_multiple_attributes(many_stimuli, crossval_f
     assert sum(len(f.x) for f in train_fixations) == (crossval_folds - val_folds - test_folds) * len(fixations.x)
 
     assert len(train_stimuli) == crossval_folds
+
+
+@pytest.mark.parametrize('attribute_name , attribute_value, invert_match', [('dva', 1, False)])
+def test_filter_stimuli_by_attribute_dva(file_stimuli_with_attributes, fixation_trains, attribute_name, attribute_value, invert_match):
+    fixations = fixation_trains[:]
+    filtered_stimuli, filtered_fixations = filter_stimuli_by_attribute(file_stimuli_with_attributes, fixations, attribute_name, attribute_value, invert_match)
+    if attribute_name == 'dva' and attribute_value == 1 and invert_match is False:
+        inds = [1]
+        expected_stimuli, expected_fixations = create_subset(file_stimuli_with_attributes, fixations, inds)
+        compare_fixations(filtered_fixations, expected_fixations)
+        assert list(filtered_stimuli.stimulus_ids) == file_stimuli_with_attributes.stimulus_ids[inds]
+        assert_stimuli_equal(filtered_stimuli, expected_stimuli)
```

Comment:
why do you need the first assertion? Isn't `assert_stimuli_equal` already taking care of that?

Reply:
ohh I was using this before I started using the create_subset function in the test itself. But now since the create_subset is already tested I will remove this line.

Reply:
fixed